JS-Algorithms/challenges.js

```diff
@@ -41,7 +49,16 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+
```

Comment:
What could happen if I send a negative number as an index here?

Reply:
Which value will be appended if index is negative?

Reply:
It will append an `undefined` at the beginning, I forgot to make that check.